Documentation/design-docs/framework-version-resolution.md

```diff
@@ -314,3 +315,37 @@ There should not be any circular dependencies between frameworks.
 A newer version of a shared framework should keep or increase the version to another shared framework (never decrease the version number).
 
 By following these best practices we have optimal run-time performance (less processing and probing) and less chance of incompatible framework references.
+
+
+## Changes to existing apps
+The above proposal will impact behavior of existing apps (because framework resolution is in `hostfxr` which is global on the machine for all frameworks). This is mostly a description of the changes as they apply to apps using either default, `rollForwardOnNoCandidateFx` or `applyPatches`.
```

Comment:
```suggestion
The above proposal will impact behavior of existing apps (because framework resolution is in `hostfxr` which is global on the machine for all frameworks). This is a description of the changes as they apply to apps using either default, `rollForwardOnNoCandidateFx` or `applyPatches`.
```